src/components/viewport/TransformToolbar.js

```diff
@@ -96,11 +97,19 @@ export default class TransformToolbar extends React.Component {
     let objects = this.state.objects;
     let objectChanges = [];
 
+    console.log(AFRAME.INSPECTOR.history.updates);
+    if (AFRAME.INSPECTOR.history.updates && Object.keys(AFRAME.INSPECTOR.history.updates).length === 0 && Object.getPrototypeOf(AFRAME.INSPECTOR.history.updates) === Object.prototype){
```

Comment:
Can you clarify this if statement a bit? What is this check for?
`Object.getPrototypeOf(AFRAME.INSPECTOR.history.updates) === Object.prototype`

Reply:
Just to check that it's empty